src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -0,0 +1,353 @@
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections;
+using System.ComponentModel;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+
+namespace System.Drawing
+{
+
+    public class ColorConverter : TypeConverter
+    {
+        private static string ValuesLock = "values";
+        private static System.ComponentModel.TypeConverter.StandardValuesCollection values;
```

Comment:
Static fields should be prefixed with s_.  I'll stop commenting on this, but it applies throughout.